src/components/auth/AuthStyles.module.scss

```diff
@@ -0,0 +1,171 @@
+@import '../../variables.scss';
+
+.auth {
+  background-color: $login-background-color;
+  height: inherit;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.authContainer {
+  margin-bottom: 1em;
+  background-color: white;
+  box-shadow: 0 10px 15px -3px rgba(0, 0, 0, 0.1), 0 4px 6px -2px rgba(0, 0, 0, 0.05);
+  padding: 2em;
+  display: flex;
+  flex-direction: column;
+  width: 300px;
+}
+.authLabel {
```

Comment:
We should be consistent on spaces between styles. Between .auth and .authConatiner there is a space and there are no space between remaining styles.